integration_tests/androidx_test/src/main/AndroidManifest-NoTestPackageActivities.xml

```diff
@@ -12,7 +12,12 @@
         <activity android:name="org.robolectric.integrationtests.axt.EspressoActivity"
                   android:label="Activity Label"
                   android:exported="true" />
+        <activity android:name="org.robolectric.integrationtests.axt.NoActionBarEspressoActivity"
+            android:label="Activity Label"
+            android:theme="@style/Theme.AppCompat.NoActionBar"
+            android:exported="true" />
         <activity android:name="org.robolectric.integrationtests.axt.EspressoScrollingActivity"
+            android:theme="@style/Theme.AppCompat.NoActionBar"
```

Comment:
Since you created a dedicated `Activity` without an `ActionBar`, is this change still necessary?
Same for the other modified manifests.